src/config.coffee

```diff
@@ -145,6 +146,7 @@ module.exports = class Config extends EventEmitter
 			deltaApplyTimeout: { source: 'db', mutable: true, default: '' }
 			deltaRetryCount: { source: 'db', mutable: true, default: '30' }
 			deltaRetryInterval: { source: 'db', mutable: true, default: '10000' }
+			deltaVersion: { source: 'db', mutable: true, default: 2 }
```

Comment:
If source is 'db', the default should be a string (and then keep in mind that the value will always be a string).
Might actually be good to add a comment with this rule on top of the schema...